src/js/App/Header/Loader.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const HeaderLoader = () => <React.Fragment />;
+
+export default HeaderLoader;
```

Comment:
You can just export  `React.Fragment`. No need for additional node in the VirtualDOM.

Reply:
I wanted to use some fancy loader similiar to navigation, but had hard time doing it. Let's nuke this file and use just Fragment there.